### PR TITLE
Adds the HF Blade, removes the unused one from the code

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
it dismembers and that's it
I also removed the old adminspawn only one
## Changelog
:cl:
add: think something up funny for me I am lazy
/:cl:

### CODE REVIEW

File: code/game/objects/items/holy_weapons.dm
Comment (EdgeLordExe) [inline code/game/objects/items/holy_weapons.dm:410]: What

File: code/game/objects/items/melee/hf.dm
Comment (EdgeLordExe) [inline code/game/objects/items/melee/hf.dm:40]: Not sure about this. Id accept 75% but 100 is a bit much

File: code/game/objects/items/melee/hf.dm
Comment (EdgeLordExe) [inline code/game/objects/items/melee/hf.dm:72]: This may have been changed with wounds

File: code/modules/uplink/uplink_items.dm
Comment (EdgeLordExe) [inline code/modules/uplink/uplink_items.dm:2135]: Way too low. Make it at least 15 tc

**Review by EdgeLordExe — CHANGES_REQUESTED** [review]
Balancing concerns

File: code/game/objects/items/holy_weapons.dm
Comment (Shadowfgt) [reply code/game/objects/items/holy_weapons.dm:410]: They probably changed the name so it would not be confused with the real thing and get a chap lynched

Comment (karmaisblackandbluepilled) [reply code/game/objects/items/holy_weapons.dm:410]: spot on

File: code/modules/uplink/uplink_items.dm
Comment (karmaisblackandbluepilled) [reply code/modules/uplink/uplink_items.dm:2135]: ok

File: code/game/objects/items/melee/hf.dm
Comment (karmaisblackandbluepilled) [reply code/game/objects/items/melee/hf.dm:72]: yeah hold on lemme test if that's true
somehow
uh

File: code/game/objects/items/melee/hf.dm
Comment (karmaisblackandbluepilled) [reply code/game/objects/items/melee/hf.dm:40]: sleeping carp gives you 100%

File: code/game/objects/items/melee/hf.dm
Comment (karmaisblackandbluepilled) [reply code/game/objects/items/melee/hf.dm:72]: ok I understand how to test now

**Comment by AcapitalA** [comment]
If you're gonna do this, at least do something with the old HF blade, such as removing it.

**Comment by karmaisblackandbluepilled** [comment]
the old hf blade is a nullrod, why would I remove it

**Comment by karmaisblackandbluepilled** [comment]
oh I see what you're talking about
why would I remove that one, it's an actual weapon that deals damage instead of dismembering

**Comment by AcapitalA** [comment]
It's just weird that there's 2 weapons with the same name and identical icons are in the game, it'll also be confusing for admins and players alike considering the block chances of both.

**Comment by cuckooman** [comment]
someone did alternate sprites for Sam's sword - that could be used for one of them
https://media.discordapp.net/attachments/878737577748856832/957737593561284679/unknown.png

**Comment by cuckooman** [comment]
for more differentiation

**Comment by AcapitalA** [comment]
> someone did alternate sprites for Sam's sword - that could be used for one of them https://media.discordapp.net/attachments/878737577748856832/957737593561284679/unknown.png

Agreed, best solution right here. Hell, maybe even special interactions for both.

**Comment by karmaisblackandbluepilled** [comment]
are you both retarded

**Comment by cuckooman** [comment]
> are you both retarded

quote possibly

**Comment by AcapitalA** [comment]
what

File: code/game/objects/items/melee/hf.dm
Comment (TheChosenEvilOne) [inline code/game/objects/items/melee/hf.dm:44]:
```suggestion
		if(ishuman(hitby.loc))
			var/mob/living/carbon/human/attacker = hitby.loc
			if(prob(20) || HAS_TRAIT(attacker, TRAIT_CLUMSY))
```

File: code/game/objects/items/melee/hf.dm
Comment (TheChosenEvilOne) [inline code/game/objects/items/melee/hf.dm:63]:
```suggestion
		for(var/X in C.bodyparts)
			var/obj/item/bodypart/BP = X
			if(!BP.dismemberable)
				continue
			if(!(BP.body_part & (HEAD | CHEST)))
				parts += BP
```

**Comment by JinGiant** [comment]
Give me maintainer so I can add the do not merge tag
This pr fucking sucks

**Comment by karmaisblackandbluepilled** [comment]
ok so I think I made it have different block changes based on if you're wielding it or not.
33 melee 80 ranged if wielded
80 melee 20 ranged if not

**Pull request merged**
